go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/openshift/cluster-svcat-controller-manager-operator
+
+go 1.13
+
+require (
+	github.com/openshift/api v0.0.0-20200217161739-c99157bc6492
+	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
+	github.com/sirupsen/logrus v1.4.2
+	k8s.io/apimachinery v0.17.3-beta.0
+	k8s.io/client-go v0.17.2
```

Comment:
Are we using these versions in SDK?

Reply:
@camilamacedo86 not that I know of. But this operator doesn't use or require the SDK. So there should be no conflict between the two.